modules/twinkleblock.js

```diff
@@ -1040,6 +1040,14 @@ Twinkle.block.blockPresetsInfo = {
 		reasonParam: true,
 		reason: 'Email [[WP:Harassment|harassment]]',
 		summary: 'You have been [[WP:PB|blocked]] from emailing other editors for [[WP:Harassment|harassment]]'
+	},
+	'uw-aepblock': {
+		autoblock: true,
+		nocreate: false,
+		pageParam: false,
+		reason: '[[WP:Arbitration enforcement|Arbitration enforcement]]',
```

Comment:
make this custom, AE is too vague.

Reply:
What to?  This is the same as `ue-aeblock`, and it's just the default in the box; sysops can of course change.

Reply:
> What to? This is the same as `ue-aeblock`, and it's just the default in the box; sysops can of course change.
Add a tooltip here then? I stated this actually because I got a query regarding which AE did I protect the pages under (albeit my responsibility).

Reply:
That'd be a good thing for the DS box to add, but yeah, a generic "You may want to change this from the default" tooltip couldn't hurt.